llmfoundry/models/inference_api_wrapper/openai_causal_lm.py

```diff
@@ -228,9 +228,14 @@ def process_result(self, completion: Optional['ChatCompletion']):
             tensors = []
             for t in self.tokenizer(
                     completion.choices[0].message.content)['input_ids']:
-                tensors.append(
-                    self.tokenizer.construct_logit_tensor(
-                        {self.tokenizer.decode([t]): 0.0}))
+
+                # Construct tensor of shape (vocab_size,) with logprobs for each token
```

Comment:
From https://github.com/mosaicml/llm-foundry/blob/ac7835415cf6fa91664e513c7a471f2802cdc8a2/llmfoundry/tokenizers/tiktoken.py#L361

Reply:
Should this fix just be in `tiktoken.py`? or why are you copying it over here?

Reply:
nvm